assets/favicons/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Boosted",
```

Comment:
`name` and `short_name` should be modifiable by the projects using this theme. If not possible maybe have something more generic like, IDK, "Orange documentation" (something like that)